modules/openid_auth/app/controllers/openid_auth/v2/validation_controller.rb

```diff
@@ -10,8 +10,6 @@ class ValidationController < ApplicationController
 
       def index
         render json: validated_payload, serializer: OpenidAuth::ValidationSerializerV2
-      rescue => e
-        raise Common::Exceptions::InternalServerError, e
```

Comment:
I'd like to get @dbrown-va to get some eyes on this. I'm not sure how this affects unhandled/runtime errors.

Reply:
This part is changed in in the part2 changes.

Reply:
I think this removal will allow for unexpected exceptions to be raised in a less controlled fashion.

Earlier in the PR thread I see there was the `e.detail.eql?('Invalid request')` logic inside the rescue block.

Checking out the branch now to take a look at the different scenarios that could occur.

Reply:
That original one would need to check e.detail.is_nil? if it were still there.

Reply:
I think checking for the `Common::Exceptions::TokenValidationError` error type which is set with the `error_klass` method would provide the most consistency.  Then ensure any other unexpected errors are returned at least as the InternalServerError rather than a stack trace.